test/activity_test.go

```diff
@@ -105,9 +105,7 @@ func (a *Activities) invoked() []string {
 	a.mu.Lock()
 	defer a.mu.Unlock()
 	result := make([]string, len(a.invocations))
-	for i := range a.invocations {
-		result[i] = a.invocations[i]
-	}
+	copy(result, a.invocations)
```

Comment:
👍 though I do look forward to generics-izing this kind of thing.  seems a shame to leave the "set wrong len / did not check return value" gap exposed.

Reply:
static check was highlighting this so was obvious fix